spec/support/factory_girl.rb

```diff
@@ -12,10 +18,11 @@
     venue_name { Faker::Hipster.word }
     description { Faker::Hipster.paragraph }
     street_name { Faker::Address.street_address }
-    state { Faker::Address.state }
+    state { STATE_ARRAY.sample }
     zip_code { Faker::Address.zip[0..4] }
     city { Faker::Address.city }
     price_range "2"
+    user
   end
```

Comment:
Using faker to seed data in development makes sense. Typically, you do not want random data in your test suite. This can create difficult-to-reproduce errors.